marimo/_runtime/dataflow/graph.py

```diff
@@ -412,4 +534,24 @@ def definitions(self) -> Mapping[Name, set[CellId_t]]:
     @property
     def cycles(self) -> set[tuple[tuple[CellId_t, CellId_t], ...]]:
         """Get the cycles set."""
-        return self.cycle_tracker.cycles
+        return self.topology.cycles
+
+
+# Used over _MarimoGraph for backwards compatibility
+class DirectedGraph(_MarimoGraph):
```

Comment:
This feels excessive to me. I would recommend renaming `_MarimoGraph` to just be `DirectedGraph` as it was before